app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  LENGTHEMAIL = (6..50).to_a.freeze
```

Comment:
Constant should have readable name with underscore between separate words, like `EMAIL_LENGTH`